x-pack/plugins/uptime/public/components/fleet_package/types.tsx

```diff
@@ -115,6 +115,9 @@ export enum ConfigKeys {
   TLS_VERSION = 'ssl.supported_protocols',
   TAGS = 'tags',
   TIMEOUT = 'timeout',
+  DOWNLOAD_SPEED = 'throttling.download_speed',
+  UPLOAD_SPEED = 'throttling.upload_speed',
+  LATENCY = 'throttling.latency',
```

Comment:
We may want to rethink putting this into 3 values. Here's why:

This works well for the Elastic Synthetics integration with Fleet. We have multiple layers of formatting to format fields in a way that fleet and the heartbeat will understand. The formatting flow is.

Raw UI format (the one we use for the components) -> UI formatters -> fleet -> package `${datastream}.yml.hbs` file.

Right now the `.yml.hbs` file is doing the work of formatting this into viable heartbeat.yml format. However, in the new UI monitor management flow we won't have that additional formatting layer.

We can add one, of course, but it's something to consider. In the past, we've made configuration keys that logically are multiple fields to a user, but one field in heartbeat, an object with multiple keys. We've then used our UI formatters to format that into one value to pass to fleet. 

I thought that this choice worked well when we were just considering fleet. But now that we'll be interacting with heartbeat directly, we'll either need to make this one config key that we store as an object in the UI but format as a string in our formatters, or we need to consider adding another layer of formatting to remove unused fields or merge fields.

Reply:
That's an excellent suggestion. I 100% agree.

Considering we'll now be interacting with `heartbeat` directly it would indeed be good to have one fewer layer of formatting for these values.

To be philosophical:

> _Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away._
> _Antoine de Saint-Exupéry, Airman's Odyssey_

I'll update the PR right now :)